### DISCUSSION TITLE
Is it possible to use ServiceBus Topic and Subscription

### DISCUSSION THREAD

**Discussion by jitendrabehera2** [kedacore/keda#1111] [Q&A / Need Help] [answered]
Hi,
I want to scaling implement using serviceBus topic and subscription. 
Below is the configuration i have used but its not working in my case.
Could you please suggest what i have missed or what anything need to do.

For Create Secret

apiVersion: v1
kind: Secret
metadata:
  name: order-secrets
  labels:
    app: order-processor
data:
  SERVICEBUS_QUEUE_CONNECTIONSTRING: <base64 connection  string >

-----------------------

For Deployment
-----------
apiVersion: apps/v1
kind: Deployment
metadata:
  name: order-processor
  labels:
    app: order-processor
spec:
  selector:
    matchLabels:
      app: order-processor
  template:
    metadata:
      labels:
        app: order-processor
    spec:
      containers:
      - name: order-processor
        image: containerregistryprismpoc.azurecr.io/qc_fl_dade:latest
        env:
        - name: KEDA_SERVICEBUS_QUEUE_CONNECTIONSTRING
          
---
apiVersion: keda.k8s.io/v1alpha1
kind: ScaledObject
metadata:
  name: order-processor-scaler
  labels:
    app: order-processor
    deploymentName: order-processor
spec:
  scaleTargetRef:
    deploymentName: order-processor
  minReplicaCount: 1
  #Change to define how many minimum replicas you want
  maxReplicaCount: 10
  triggers:
  - type: azure-servicebus
    metadata:
      #queueName: orders
      topicName: xxxxxxxxxxxxxx
      subscriptionName: order-processor_Subscription
      connection: KEDA_SERVICEBUS_QUEUE_CONNECTIONSTRING
      queueLength: '5'

**Comment by tomkerkhove** [comment, marked as answer]
You most definitely can, what issue are you facing?

**Reply by jitendrabehera2** [reply]
Ok, its not scaling even there has message in my topic.

**Reply by jitendrabehera2** [reply]
Is my YAML configuration correct?

**Reply by tomkerkhove** [reply]
Can you share your KEDA logs please? What permissions does the connection string have?

**Reply by jitendrabehera2** [reply]
Sorry for disturbed you, Its scaling now and i got what i have missed.

Thanks for replied.

**Reply by tomkerkhove** [reply]
No worries at all! Feel free to mark this as answered.
